### PR TITLE
Update: Add deprecation warning.

### PR DESCRIPTION
**Pull request by patrick-ivers**
#### Changes
- Add deprecation warning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-alexandria
+<span style="color:red">__DEPRECATED__</span>
+
+<span style="color:red">This package is deprecated. All Alexandria components have been moved to Adslot UI.</span>
```
Comment (vinteo) [inline README.md]: add a link to adslot-ui?

**Comment by sonhanguyen** [comment]
LGTM

**Comment by patrick-ivers** [comment]
@xiaofan2406 Will do once it's merged.

**Comment by omgaz** [comment]
Merging 🍜

**Pull request merged**